relayer/naive-strategy.go

```diff
@@ -177,20 +182,26 @@ func sendTxFromEventPackets(src, dst *Chain, rlyPackets []relayPacket, sh *SyncH
 }
 
 // RelayPacketsUnorderedChan creates transactions to relay un-relayed messages
-func (nrs *NaiveStrategy) RelayPacketsUnorderedChan(src, dst *Chain, sp *RelaySequences, sh *SyncHeaders) error {
+func (nrs NaiveStrategy) RelayPacketsUnorderedChan(src, dst *Chain, sp *RelaySequences, sh *SyncHeaders) error {
 	// TODO: Implement unordered channels
 	return nrs.RelayPacketsOrderedChan(src, dst, sp, sh)
 }
 
 // RelayPacketsOrderedChan creates transactions to clear both queues
 // CONTRACT: the SyncHeaders passed in here must be up to date or being kept updated
-func (nrs *NaiveStrategy) RelayPacketsOrderedChan(src, dst *Chain, sp *RelaySequences, sh *SyncHeaders) error {
+func (nrs NaiveStrategy) RelayPacketsOrderedChan(src, dst *Chain, sp *RelaySequences, sh *SyncHeaders) error {
```

Comment:
Ok, so what is the behavior here now? This function needs to be re-run until there are no more messages to send?

Reply:
yea, I was thinking it'd be nice to have a func that pulls from the queue and creates the tx's based on the max params, in case you wanna just send one of those. On second thought a call to this func should clear the passed in queues. I can refactor this code some more so it clears the passed in relay seq's in one call